src/geom.rs

```diff
@@ -929,4 +929,12 @@ mod rect_tests {
         assert_eq!(rect.width(), 20.0);
         assert_eq!(rect.height(), 20.0);
     }
+
+    #[test]
+    fn issue_49() {
+        let specific_value = f32::from_be_bytes([0x64, 0x00, 0x00, 0x00]);
+        let rect = Rect::from_xywh(300.0, -1.0, specific_value, 1.0).unwrap();
```

Comment:
`let rect = Rect::from_xywh(1.0, 0.0, i32::MAX as f32, 1.0).unwrap();`

I think this would be more obvious. Or something along those lines